RevenueCatUI/Templates/Example1Template.swift

```diff
@@ -71,17 +61,25 @@ private struct Example1TemplateContent: View {
     @ViewBuilder
     private var content: some View {
         VStack {
-            Image("image", bundle: .module)
-                .resizable()
-                .frame(maxWidth: .infinity)
-                .aspectRatio(1, contentMode: .fill)
-                .edgesIgnoringSafeArea(.top)
-                .padding(.bottom)
-                .mask(alignment: .top) {
-                    Circle()
-                        .offset(y: -160)
-                        .scale(2.5)
+            AsyncImage(url: self.data.headerImageURL) { phase in
```

Comment:
this is fine for now, but I think sooner or later we'll have to move into a system that pre-downloads and caches the image. Paywalls need to have the smallest possible wait time to load

Reply:
Yup 👍🏻